java/src/main/java/vowpalWabbit/learner/VWLearner.java

```diff
@@ -10,4 +11,9 @@
 
     // This tells the implementations that an IOException cannot be thrown.
     @Override void close();
+
+    void saveModel(File filename);
+    
+    // Get the JNI pointer for the learner
+    public long getNativePointer();
```

Comment:
That line got brought in when I merged master in to resolve the previous merge conflict. I have no idea why it was added. And of course, it then got removed from master, creating yet another merge conflict.